### PR TITLE
Remove legacy integrations

### PR DESCRIPTION
**Pull request by jasonaowen**
There were three integrations apparently built out for Minnesota: exports via FileNet, exports via JMS, and imports via JMS. Remove them, so as to make it easier to delete some of the yet to be converted (and, perhaps, already converted) entities, and leave reminder comments to add hooks for issue #215.

Note that while we no longer need to configure a message queue, we do still need the `standalone-full` WildFly profile: trying to deploy on the `standalone` profile failed.

Issue #36 Use Hibernate 5, instead of 4
Issue #215 Add extension points to allow per-state integration

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -217,15 +211,6 @@ If you are using a production mail server, add a mail session with a JNDI name
...
 
-### Messaging
-
-Create a messaging queue:
-
-```ShellSession
-$ ./bin/jboss-cli.sh --connect \
-  --command='jms-queue add --queue-address=DataSync --entries=["java:/jms/queue/DataSync"]'
-```
-
```
Comment (cecilia-donnelly) [inline INSTALL.md]: Thanks for updating the INSTALL file as you go!

Comment (cecilia-donnelly) [inline INSTALL.md]: Thanks for keeping INSTALL up to date as you go!

Comment (cecilia-donnelly) [inline INSTALL.md]: Oh, I thought I lost this one but instead just said the same thing twice.

Comment (BenGalewsky) [inline INSTALL.md]: This looks like a good change - no need to keep these unknown legacy connections around. I don't see any problems.

**Pull request merged**